src/js/letters/containers/LetterList.jsx

```diff
@@ -6,10 +6,15 @@ import CollapsiblePanel from '../../common/components/CollapsiblePanel';
 import DownloadLetterLink from '../components/DownloadLetterLink';
 import VeteranBenefitSummaryLetter from './VeteranBenefitSummaryLetter';
 
+import { focusElement } from '../../common/utils/helpers';
 import { letterContent, bslHelpInstructions } from '../utils/helpers';
 import { AVAILABILITY_STATUSES, LETTER_TYPES } from '../utils/constants';
 
 export class LetterList extends React.Component {
+  componentDidMount() {
```

Comment:
Sam accessibility guru's recommendation was totally in line with what you did here - basically, for the second page of the app, focus on the beginning of the change, which is the 'step 2 of 2' `.nav-header`. This worked really nicely when I tried it out! 👍